README.md

```diff
@@ -1,92 +1,115 @@
-# Mandel Contracts
+# System Contracts
 
-## Repo organization
+System contracts are a collection of contracts deployable to an EOSIO blockchain that implement a lot of critical functionality that is expected from blockchains. For example, the concept of the core token of the blockchain is introduced through the system contracts. In addition, the system contracts can build upon primitives provided by the base EOSIO protocol to enable more useful capabilities, such as:
```

Comment:
I'm curious if there is a way we can be more specific about a classification of blockchains (e.g. modern blockchains, advanced blockchains) due to the fact that I feel the blanket statement of what is expected of all blockchains may be misleading. It is also an opportunity to affirm the quality and status of EOSIO blockchains.